File: src/main/java/com/webapp/dto/ContactDto.java
package com.webapp.dto;

import com.webapp.model.Contact;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import java.io.Serializable;

@Getter
@Setter
@NoArgsConstructor
@ToString
public class ContactDto implements Serializable {

    private String personId;
    private String id;
    private String type;
    private String data;
    private String description;

    public ContactDto(Contact contact) {
        this.personId = contact.getPersonId();
        this.id = contact.getContactId();
        this.type = contact.getType().ordinal() + "";
        this.data = contact.getData();
        this.description = contact.getDescription();
    }
}

File: addressbook-common/src/main/kotlin/com/addressbook/dto/ContactDto.kt
package com.addressbook.dto

import com.addressbook.model.Contact
import java.io.Serializable

class ContactDto constructor(contact: Contact? = null) : Serializable {
    var personId: String? = null
    var id: String? = null
    var type: String? = ""
    var data: String? = null
    var description: String? = null

    init {
        this.personId = contact?.personId
        this.id = contact?.contactId
        this.type = "" + contact?.type?.ordinal
        this.data = contact?.data
        this.description = contact?.description
    }
}
